src/current/v23.2/backup-validation.md

```diff
@@ -15,13 +15,17 @@ You can validate a backup of a [cluster]({% link {{ page.version.version }}/back
 
 The options that give the most validation coverage will increase the runtime of the check. That is, `verify_backup_table_data` will take a longer time to validate a backup compared to `check_files` or `schema_only` alone. Despite that, each of these validation options provide a quicker way to validate a backup over running a "regular" restore.
 
+{{site.data.alerts.callout_info}}
+You do **not** need an [{{ site.data.products.enterprise }} license]({% link {{ page.version.version }}/enterprise-licensing.md %}) to run any of the backup validation tools described on this page.
+{{site.data.alerts.end}}
+
 ## Recommendations
 
 Cockroach Labs recommends implementing the following validation plan to test your backups:
 
-1. Very frequent [`schema_only`](#validate-a-backup-is-restorable) restores: Checks your credentials, schema issues, and version compatability.  
-1. Frequent [`verify_backup_table_data`](#validate-backup-table-data-is-restorable) restores: Tests that all data files are present and uncorrupted. 
-1. Somewhat frequent ["full"]({% link {{ page.version.version }}/restore.md %}) restores: Validates a backup completely. 
+1. Very frequent [`schema_only`](#validate-a-backup-is-restorable) restores: Checks your credentials, schema issues, and version compatability.
```

Comment:
minor spelling error: compatability should be compatibility

Reply:
whoooops! Thanks for the catch on that Florence!